docs/api.mdx

```diff
@@ -64,11 +65,19 @@ This is useful for sending multiple transactions that depend on each other.
 
 **Refunds:**
```

Comment:
```suggestion
**MEV Refunds:**
```
Let's use the terminology we already use [here](https://docs.flashbots.net/flashbots-protect/mev-refunds)